File: ping.js
const ping = require('ping');

const sites = [
    'fastsong.eu.org',
    'www.fastsong.eu.org',
    // Add 98 more active search sites here
];

const pingSites = async () => {
    for (const site of sites) {
        try {
            const res = await ping.promise.probe(site);
            console.log(`${site}: ${res.alive ? 'Alive' : 'Dead'}`);
        } catch (error) {
            console.error(`Error pinging ${site}: ${error.message}`);
        }
    }
};

pingSites();

File: ping.py
import os
import subprocess

def ping_sites(sites):
    for site in sites:
        response = subprocess.run(['ping', '-c', '1', site], stdout=subprocess.PIPE)
        print(f'Pinging {site}:')
        print(response.stdout.decode())

if __name__ == "__main__":
    sites_to_ping = ['fastsong.eu.org'] * 100 + ['www.fastsong.eu.org'] * 100
    ping_sites(sites_to_ping)
